Makefile

```diff
@@ -496,23 +495,25 @@ ifneq ($(shell mdbook --version > /dev/null 2>&1 && echo $$?), 0)
 	@echo -e "    cargo +stable install mdbook-linkcheck --force"
 	@exit 1
 endif
-	@mdbook build $(BOOK_SRC)
-	@rm -rf $(BOOK_OUT)
-	@mkdir -p $(BOOK_OUT)
-	@cp -rf $(BOOK_SRC)/book/html/* $(BOOK_OUT)
+	@mdbook build $(BOOK_SRC) -d $(BOOK_OUT)
```

Comment:
Building the book directly to the output directory makes the build command much shorter but adds unneeded nesting to the published output. Since the entire `docs/` directory is published to GitHub Pages the final output for the book would be https://theseus-os.github.io/Theseus/book/html/index.html. The `html/` nesting is avoided by the copy. Another drawback to building directly is that the `linkcheck/` directory is also published to GitHub Pages.

Reply:
Good points. I was only considering the local build process when making these changes, not the published Pages actions. Oops.